main/tests.py

```diff
@@ -82,8 +81,20 @@ def test_root_entrypoint_redirection_logged_user(self, client, create_dog_owner_
         assert response['Location'] == '/homepage/'
 
 
+@pytest.mark.django_db
 class TestHomepageView:
     def test_unlogged_user_access_to_homepage(self, client):
         response = client.get("/homepage/")
         assert response.status_code == 302
         assert response['Location'] == '/login/?next=/homepage/'
+
+    def test_dog_owner_homepage_is_visible_for_dog_owner(self, client, create_dog_owner_user):
+        client.force_login(user=create_dog_owner_user.user)
+        response = client.get("/homepage/")
+        assert response.status_code == 200
+        assert response.context['daycares']
```

Comment:
what are we testing here?

Reply:
We are testing successful enter to dog_owner_home page and that the response containing 'daycares' sent from dog_owner_home view .

Reply:
If I understand correctly,  here we only test that the HTML has daycares argument, isn't it redundant?
because this argument surely gets passed from views.py. 
Maybe it's better to check the value (the logic in views.py) of this argument is really what we expect

Reply:
1) Im not sure it is redundant because if the user is day_care he will not have daycares in response.context -
so in that way I can make sure that the current URL is in dog_howner_home.
2)Can you share what r u thinking about ? the logic in the home_page_view for now is to send all daycares with the request.

Reply:
I agree, we should not only check the existence of "daycares" in the context, but the daycares information in the context as well.

Reply:
Added a check for the context of the "daycares" from the response.